batchflow/batch.py

```diff
@@ -1089,3 +932,31 @@ def dump(self, *args, dst=None, fmt=None, components=None, **kwargs):
     def save(self, *args, **kwargs):
         """ Save batch data to a file (an alias for dump method)"""
         return self.dump(*args, **kwargs)
+
+    @apply_parallel_
+    def to_array(self, comp, dtype=None, channels='last'):
```

Comment:
Maybe `float32` as default `dtype`? This action already has a strong flavour of ML, so it sounds reasonable